### PR TITLE
Update Get-ADFSEndpoint.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #1882 - Example Correction.

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
I'm OK with the change itself when the build block is fixed.

**Comment by sowjanya365** [comment]
Hi @get-itips 

Thank you! I've updated the PR now. Kindly verify now. Thanks.

File: docset/winserver2012-ps/adfs/Get-ADFSEndpoint.md [hunk 1]
```diff
@@ -32,18 +32,12 @@ The collection of ADFSEndpoint objects is a list of all the supported endpoints
 You can use this list to view the configuration of endpoints and enable or disable them.
 To see the full list of endpoints, use this cmdlet with no parameters.
 
-## EXAMPLES
-
-### -------------------------- EXAMPLE 1 --------------------------
+## EXAMPLES: Get an endpoint
```
Comment (get-itips) [inline docset/winserver2012-ps/adfs/Get-ADFSEndpoint.md:35]:
```suggestion
## EXAMPLES
```

**Comment by sowjanya365** [comment]
Thank you, @get-itips  :)

**Comment by get-itips** [comment]
ok let's see if the build passes now..

**Pull request merged**